src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -506,9 +559,21 @@
 type: command
 short-summary: Add a custom registry login credential to the task
 examples:
-  - name: Add a registry login credential to a task.
+  - name: Add a registry login credential to a task using a plain text username and password.
     text: >
         az acr task credential add -n taskname -r registryname --login-server myregistry.docker.io -u 'myusername' -p 'mysecret'
```

Comment:
Curious, can we actually reference a docker hub registry by `myregistry.docker.io`?

Reply:
it will work with `--username`, `--password`, `--vault-username` or `--vault-password`. 


Not with MSI, bc with MSI we have a special process of getting ARM token and exchanging it with ACR token and then `docker login reg -u 000000 -p $token`

Reply:
@shahzzzam just to clarify it is `--kv-username` and `kv-password`